### PR TITLE
Senior Roles

### PR DESCRIPTION
**Pull request by Nairodian**
## About the PR
Adds four new roles, Senior Engineer, Senior Researcher, Senior Physician, and Senior Officer.
These are designed to act as a guiding role for interns and staff members alike, as heads are usually too bogged down by station concerns to handle that themselves. In cases where it's required, a senior can act as a second in-command of a department as they would be the most experienced.

Seniors are given no extra gear that the normal staff member couldn't get themselves, as to avoid people using it just for the extra gear.
All the senior roles are playtime locked. They require double the playtime that the related head does, which is 60 hours overall in their department.

**Media**
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**
:cl:
- add: Added four new senior roles meant for teaching new players their related department; Senior Engineer, Senior Researcher, Senior Physician and Senior Officer.

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
This 100% should be a loadout thing.

**Comment by Nairodian** [comment]
Well, we don't have loadouts. This can be properly merged into the loadout system when that becomes a thing.

**Comment by Emisse** [comment]
hey this isnt in loadout

**Comment by Skarletto** [comment]
mentor lite

**Comment by LankLTE** [comment]
I could see some issues arising with them being unable to be traitors, but I do get the logic behind it so it’s probably best to let time tell if there’ll actually be issues. 

~~Cargo getting shafted again.~~

**Comment by Just-a-Unity-Dev** [comment]
i do like the uniforms

**Comment by Nairodian** [comment]
> ~Cargo getting shafted again.~

Cargo just isn't advanced enough to warrant having a senior, the QM can usually manage it on his own.

**Comment by Vaaankas** [comment]
We already have the newbie roles (which i believe were doubled in amount in #17440), regular staff and heads, Sec even has Warden which is kind of a senior. I honestly dont see a point in adding another step to the hierarchy.

Also every department having another character thats unable to be an antag is just bad for antags.

**Comment by EmoGarbage404** [comment]
Sprites are cool but making jobs just to show "this player has skill" is really bloaty and could easily just be part of a loadout system. As it stands its just adding jobs which have absolutely no mechanical difference.

**Comment by Vaaankas** [comment]
If a player wants to play a mentor role, they can do it already, all they need to do is ask their department head for a cool beret.

**Comment by Nairodian** [comment]
Heads are too busy dealing with over watching the department, and department staff rarely go to teach players, this is a role that's designed to make up for those two faults. Being quite noticeable to new players with a set ID and allowing the mentors not to have to hassle with making themselves known to the new players.

**Comment by LankLTE** [comment]
> As it stands its just adding jobs which have absolutely no mechanical difference.

I mean, technically there's one in that they can't be antags. I do get the niche the role is trying to fill, and I think there's a point to be made that heads really don't have time to teach people while interns have no (other) person to go to for help. That said I don't really see a different solution for this past like, adding a "Station Mentor" role or something. 

> Also every department having another character thats unable to be an antag is just bad for antags. 

4 confirmed non-antags isn't a _huge_ deal (there used to be 3 in atmos which generally worked out fine), but I think it's most notable in that people can now be _certain_ their chosen replacement for a dead head isn't an antag, and it makes sense IC. Will make it 10x more difficult for syndies to ever be promoted to a head role.

**Comment by Skarletto** [comment]
Ideally we'd have mentorhelp but I do like the idea of this
I've had many times where I had to teach players in completely different departments than mine because no one working in theirs would help them

**Comment by metalgearsloth** [comment]
I think I mentioned this when you asked me but I would drop interns and replace with seniors if we go this route.

**Comment by Nairodian** [comment]
> I think I mentioned this when you asked me but I would drop interns and replace with seniors if we go this route.

Yeah, I wanted to first get the senior roles in first, see how it's handled and later, if required, pull intern roles.

**Comment by Vaaankas** [comment]
I dont think the benefits of having mentors outweighs the downsides stated above, this will also inevitably cause the power tension between the head and the senior, especially since the senior requires more hours to unlock. I think we should promote regular department members stepping up to teach interns or free some time for the heads in some other ways.

**Comment by Nairodian** [comment]
I'm unsure how you would even sway the community to do that, there are people who try already, however they're often few and far between, I was hoping having a set role for it would draw some attention towards it.

Freeing time off the head's hands is nigh impossible at the game's current state, things are likely only going to get more stressful for them.

And removing their syndicate block would likely break the whole point of the roles.

**Comment by LankLTE** [comment]
> I'm unsure how you would even sway the community to do that, there are people who try already, however they're often few and far between, I was hoping having a set role for it would draw some attention towards it.

This is sort of how I feel. Someone going out of their way to teach people is uncommon- it's generally considered "the job of the head". Having a role designated to do it will make people do that job more often.

**Comment by RyanStrudfelt** [comment]
Personally I think having roles intended to be more dedicated to helping newer players learn would be a great boon. 

In the case of security for example: the head of security is usually stretched pretty thin dealing with both command issues, and organising the regular officers into action. 

Whilst this is going on, usually the warden is tasked with operating the brig, setting times for prisoners, guarding the armoury and sometimes even dealing with issues in perma.
 
I've seen a more than a few rounds where wardens just couldn't spend time teaching cadets, because they were picking up slack for the head of security. 

In terms of engineering, this frees up the CE, and atmos techs to deal with life threatening issues, whilst the senior engineer teaches the technicians how to avoid gassing the crew or how to un-shock a door / beginning to deal with riskier methods of power generation. 

Senior researcher would even allow for the RD to focus on one side of things, such as anomalous research and teaching research assistants one half of the job, while the senior researcher teaches the other half of the job in xenoarcheology. 
Or vice-versa. 

I'm not even going to repeat myself for medical. We all know how chaotic Med can get. 

I also don't think these roles being antag safe is a bad idea, as they could in many cases function as the next head of department should the unthinkable (and very likely) happen.
 
It preserves the trust between students and teachers, and gives the rest of command someone to turn to if a head winds up a pile of organs and blood or catatonic. 

My two cents as someone who's spent a fair bit of time on both sides of the equation across ss13/ss14.

**Comment by Vaaankas** [comment]
I dont want to be rude, but at that point you might aswell just add a second head to every department. I believe the game is meant to be chaotic, that's the whole point. And in chaos rise players willing to take action and responsibility, it doesnt take an extra role to step up and teach interns. it doesnt take an extra role to organize the department if your head is dead or missing. And having an extra antag-free role in every department will promote antags murderboning instead of trying to infiltrate the command ranks, and this in my opinion is just boring and bad for the game.

**Comment by UbaserB** [comment]
I actually like this a lot except for one thing. Not having better starting loot is fine, but id like some differences in what they start with. For example, how paramedic has a lot more emergency medipens.

**Comment by Nairodian** [comment]
> I dont want to be rude, but at that point you might aswell just add a second head to every department. I believe the game is meant to be chaotic, that's the whole point. And in chaos rise players willing to take action and responsibility, it doesnt take an extra role to step up and teach interns. it doesnt take an extra role to organize the department if your head is dead or missing. And having an extra antag-free role in every department will promote antags murderboning instead of trying to infiltrate the command ranks, and this in my opinion is just boring and bad for the game.

The game is chaotic, but to a point where it hurts and drives off new players. People rarely take action to teach new people or step up to be a (competent) organizer of their department. And this does lessen the chance of an antag getting into a command position via non-lethal means, but they can always kill and impersonate the senior too. Not to mention if a senior was promoted to acting head they might promote a new senior to replace them.
Accessibility to new players matters. Being taught in-game and in-character is infinitely more fun, interesting and memorable.

**Comment by Nairodian** [comment]
> I actually like this a lot except for one thing. Not having better starting loot is fine, but id like some differences in what they start with. For example, how paramedic has a lot more emergency medipens.

I want to keep it as simple as possible for now, their first concern is teaching and that shouldn't require much more gear than what anyone else has available to them.

**Comment by RyanStrudfelt** [comment]
> I dont want to be rude, but at that point you might aswell just add a second head to every department. I believe the game is meant to be chaotic, that's the whole point. And in chaos rise players willing to take action and responsibility, it doesnt take an extra role to step up and teach interns. it doesnt take an extra role to organize the department if the head is dead or missing. And having an extra antag-free role in every department will promote antags murderboning instead of trying to infiltrate the command ranks, and this in my opinion is just boring and bad for the game.

I disagree, I think in a game where there is so much going on, and so often few people willing to actually sit there and explain the entire workings of a job to someone else. This would be greatly appreciated for newer players. 

It would be a good way to make the game more accessible to new players, and if someone's taking the role for it's intended purpose. You'll probably find that they're just as likely to turn down a promotion if they're busy with teaching a new player or two. 

At the end of the day, antags often do just go on a murdering spree, and little will stop that. 

It's likely that this will take some tweaking over time. As with any new addition to a game. But the premise here is promising and promoting community growth.

**Comment by UbaserB** [comment]
> Contributor

ah ok, i guess thats fine

**Comment by Skarletto** [comment]
> I actually like this a lot except for one thing. Not having better starting loot is fine, but id like some differences in what they start with. For example, how paramedic has a lot more emergency medipens.

I don't like the idea of giving them more gear mainly due to what Nairod already stated in the previous comment:

> I want to keep it as simple as possible for now, their first concern is teaching and that shouldn't require much more gear than what anyone else has available to them.

Giving them more than what a normal worker has would likely increase the 'power tension' that was feared in another comment earlier

**Comment by hord-brayden** [comment]
This is a great addition. I believe that this role will attract the kinds of folks who want to teach and set the tone for behavior moving forward.

This role is an advisor to the head, if I'm reading it correctly. The way time is calculated for roles now, means any idiot can be a head. Thereby resulting in frequent trips to med bay for incompetence (and starvation).

If it were me, Id make this role a whitelisted role. When someone's a 'senior' anything - that should mean something, almost like the idealized person for a given role. I know a few players in the community who would fill this role perfectly and find a lot of joy in it. That's the kind of people I'd want teaching new players - not some douche canoe who shoves people down disposals or feeds SSD people pills.

FWIW - well done,

**Comment by Nairodian** [comment]
The whitelist idea **IS** the best choice to go about this. However as you mentioned, admin load, as well as it's out of my expertise of coding. Now if someone ever codes mentor help these jobs can be linked up to the same whitelist as that.

**Comment by Myakot** [comment]
I absolutely love this idea for one simple reason: you can now take every single bad trait possible = still useful to the team (except if you take mute) AND can't be a castrated antag with blindness, narcolepsy and pacifism.

**Comment by hord-brayden** [comment]
> I absolutely love this idea for one simple reason: you can now take every single bad trait possible = still useful to the team (except if you take mute) AND can't be a castrated antag with blindness, narcolepsy and pacifism.

On this insightful note, removing accents from this set of roles would be ideal at some point. While the phrase "Yar, Ah ken forsooth, dinnae where be AME!" is ridiculous, imagine going into a game reading that nonsense.

**Comment by RyanStrudfelt** [comment]
> > I absolutely love this idea for one simple reason: you can now take every single bad trait possible = still useful to the team (except if you take mute) AND can't be a castrated antag with blindness, narcolepsy and pacifism.
> 
> On this insightful note, removing accents from this set of roles would be ideal at some point. While the phrase "Yar, Ah ken forsooth, dinnae where be AME!" is ridiculous, imagine going into a game reading that nonsense.

Truly, this is a valid point. 

The number of times I've had to ask someone what the hell they're saying, and I have spent a lot of time with these accents. 

Tldr. They even trip up veterans like me at times. 
I remember an accentless trait getting rejected out of PR a while back... might be worth a look into that too

**Comment by Nairodian** [comment]
If you misuse and actively make it difficult for yourself to teach people or just avoid teaching people it's likely you'll get rolebanned anyhow. (With proper warning of course)

**Comment by HoidC** [comment]
hey can I just say 60 hours is absolutely wild

can it be the same as heads 

if after 30 hours (20-30 rounds) in one department you don’t know the ins and outs of a department there’s really no hope for you learning it anyway tbh

**Comment by dmnct** [comment]
If it was possible to implement with what we have I would ask for the (lower) requirement hours to **decay** over time, so someone who hasn't even played Engineering in the last 6 months can't just jump into the "teaches engineering" role. 
As is, the very steep entry requirement will have to do for checking interest in the department

**Comment by HoidC** [comment]
> If it was possible to implement with what we have, I would ask for the (lower) requirement hours to **decay** over time, so someone who hasn't even played Engineering in the last 6 months can't just jump into the "teaches engineering" role.
> As is, the very steep entry requirement will have to do for checking interest in the department

what has significantly changed in engineering in the last 6 months

**Comment by asperger-sind** [comment]
I'm more or so fine with the role itself and the function it's trying to fulfill but having them be guaranteed non-antags is going to cause problems, people already metagame the fact that certain jobs can't be antags completely eradicating the paranoia sim that should be shrouding every player interaction.

It also removes the possibility of an antag becoming a head more diplomatically rather than killing and impersonating the previous head, because whoever is going to be promoting the engineer is going to say "you're not a senior, you might be an antag, so you're not getting promoted" even if the round lacks a senior.

Even if a senior gets promoted and decides to promote an engineer to the now free position of senior, this just removes the role timer requirement (which is imo incredibly high for what the role is) and not only that, allows a bad actor yearning for promotion to fill in the mentor role exclusively for the fact that the role is slightly above normal engineers in hierarchy.

I think this should allow antags, or just wait until loadouts are introduced and shove it there, don't think there's too much of a hurry now that the guidebook is decent and there are tutorials online.

File: Resources/Locale/en-US/job/job-supervisors.ftl [hunk 1]
```diff
@@ -6,9 +6,13 @@ job-supervisors-ce = the chief engineer
...
+job-supervisors-senior-engineering = the senior engineer and, the chief engineer
+job-supervisors-senior-medical = the senior physician and, chief medical officer
+job-supervisors-senior-science = the senior researcher and, the research director
 job-supervisors-service = chefs, botanists, the bartender, and the head of personnel
-job-supervisors-engineering = station engineers, atmospheric technicians, and the chief engineer
-job-supervisors-medicine = medical doctors, chemists, and the chief medical officer
-job-supervisors-security = security officers, the warden, and the head of security
+job-supervisors-engineering = station engineers, atmospheric technicians, the senior engineer, and the chief engineer
+job-supervisors-medicine = medical doctors, chemists, the senior physician, and the chief medical officer
+job-supervisors-security = security officers, the senior officer, the warden, and the head of security
```
[the remark this answers is not in the document]
Comment (Nairodian) [reply Resources/Locale/en-US/job/job-supervisors.ftl:16]: Yeah sure, I'll yank it from the basic department staff. It'll come naturally though, I'm sure.

**Comment by Nairodian** [comment]
> hey can I just say 60 hours is absolutely wild

Fair enough. However I likely won't be backing down on the time requirement unless a whitelist system is added instead.
Being able to manage a department is much easier than being able to efficiently _teach_ someone that department.
As well as it provides trust that this is a 100% experienced player, we don't have to worry about them breaking rules, ignoring their duty or such.

**Comment by EmoGarbage404** [comment]
i feel like this runs the risk of just being used as free slots for longtime players. I could 100% see someone wanting to play security, seeing that all the roles are filled, and taking the senior role just because they wanted to play the department. 

I also think the equating time investment to knowledge or skill isn't always correct. There are plenty of people who mindlessly get hours without a full exploration or understanding of all the mechanics. I've brought this point up in the past regarding timers: you're (at best) getting a shitter filter. You're not ensuring anyone is going to be a good mentor for new players.

Additionally, the implication that the "senior" is more experienced than the head of the department is kinda strange and a bit of an upset of the balance of the department. Giving departments yet another role that is unable to be an antagonist and is guaranteed to be experienced just drags away skilled players from roles that could really use them. It also would likely mean fewer regular promotions, since people will just metagame seniors being unable to be antagonists.

**Comment by EmoGarbage404** [comment]
i genuinely admire the positivity, but I do think that expecting people to take a senior role to automatically mean they are going to teach new players is a bit of a false hope.

For one, the only explicit thing telling players that they are supposed to teach new players is the job description, which is fairly hidden and quite up in the air as to if anyone actually reads it.

Furthermore, just by having a high role timer and being called a "senior," players that aren't aware of the intent of the role are probably just going to think it's another pseudo-head in the line of command. And while i understand banning those who do not fulfill the goal of a role (teaching other players), i think that, practically speaking, determining whether or not someone is teaching someone when they may or may not want to listen or take their advice can be rather difficult. Especially since there's not really any factors pushing interns towards senior roles.

**Comment by Mudsson** [comment]
I think this is interesting but I have significantly less faith in the ability for this to not be abused in some ways, as was demonstrated when atmos couldn't be traitors.

We will probably see what Chief Engineers did when atmos couldn't be traitor. Giving spare belt to atmos that doesn't use it over your engineers because "can't be traitor" only now it will always be a senior that gets it (honestly mostly fine so they can show newbies how to use power tools but still blocks syndies)

The CMO might stop giving their handheld monitor to paramedics and give to the senior as they're unable to be traitor

Research directors that don't use their suit might always give it to seniors, or give their teleporter once it's set up in a meta attempt to avoid having it stolen if they're killed.

Security might do some weird things where senior officers get lethals where regular officers don't, which is fine but means less opportunities for regular officers and cadets to do more interesting firefights. Security is the least problematic as they can't be traitors anyhow.


All of these make sense IC as trusting a senior  just makes sense that way, but it may block more interesting mechanics for newer players and syndicate. It would be awful if you only became able to do some of the more interesting things in departments as a senior  role, blocking new players and syndicate from these more interesting things.

**Comment by Nairodian** [comment]
> For one, the only explicit thing telling players that they are supposed to teach new players is the job description, which is fairly hidden and quite up in the air as to if anyone actually reads it.
> 
> Furthermore, just by having a high role timer and being called a "senior," players that aren't aware of the intent of the role are probably just going to think it's another pseudo-head in the line of command. And while i understand banning those who do not fulfill the goal of a role (teaching other players), i think that, practically speaking, determining whether or not someone is teaching someone when they may or may not want to listen or take their advice can be rather difficult. Especially since there's not really any factors pushing interns towards senior roles.

I very well could rename them to [department] mentor, however it doesn't feel as right; less professional, rolls off the tongue worse, and doesn't suit the rest of the naming schemes.
Another idea I had was adding a message when someone joined as the role, much akin to the adminnotify that head roles have, but that's still easily overlooked.
I think the best way it'll become known is just naturally with admin nudges, if a person sees that a certain role taught them and others, that'll stick in their mind for what that role is for.

As for pushing them together, another admin suggested making it so interns got a symbol marking senior roles, and seniors got a symbol marking interns, however I had slight concerns of metagaming and that was out of my wheelhouse. I hoped that if it became too much of a problem to return to it later and implement a system for it.

**Comment by Nairodian** [comment]
> I think this is interesting but I have significantly less faith in the ability for this to not be abused in some ways, as was demonstrated when atmos couldn't be traitors...
>
> ...Most of these are just ways heads would kinda abuse seniors, not even really their fault, and while they could refuse thse items or jobs, then they are disobeying a superior and we get more headbutting.
> 
> All of these make sense IC as trusting a senior just makes sense that way, but it may block more interesting mechanics for newer players and syndicate. It would be awful if you only became able to do some of the more interesting things in departments as a senior role, blocking new players and syndicate from these more interesting things.

This is.. fair all things considered, I hadn't thought about heads abusing the meta against seniors too much. (_god I hate the community_) I'm just mostly worried that if antag restrictions were lifted on the mentor roles it'd cause even worse issues for new players than we already have, as they can no longer trust their senior to properly teach them, causing the only trusted role to do it again being the head of the related department. Which is right back where we started.

**Comment by VMSolidus** [comment]
Here's a crazy idea for helping distinguish even to the players who play as it what the role is. Why not co-opt the antag system for this purpose? Make an "Antag" role called "Mentor" that these senior roles automatically spawn in with when they join the round, with a set objective describing their role as a trainer. Teach the interns how to do their job, and if you don't find any interns, then go about sharpening your department's skills. This way people who join in the round as a Mentor role immediately get notification of what they are meant to be doing.

**Comment by EmoGarbage404** [comment]
i like the concept a lot, but i think perhaps it'd be better to just push information to new players more? Things like a more comprehensive guidebook, a first-time popup or notifier informing them on the guidebook and wiki, and maybe even reintroducing something like Tippy the Clown could help disperse information more effectively to players.

I think any situation where you're directing players to newcomers needs to be handled sensitively, so the best way to avoid that would simply be to give new players resources to learn these things on their own, where they have full control over what information is available to them. If an in-game example is really needed, encouraging them to ask heads about it is probably good enough if paired with other options. 

(please keep the clothes tho they look baller i'll code loadouts just to have them in)

**Comment by dmnct** [comment]
What if Seniors had a REDUCED chance to be antag, so it still can't be discounted but will be less likely to be an issue for their trainees?

**Comment by VMSolidus** [comment]
Leviathan is getting pretty damn big now. A few nights ago I saw a round where my CMO had five whole interns to train, all of which had his complete attention. The CMO actually had so many people to train in a single round, that he was physically incapable of doing anything else for the entire round. Security also routinely has 4 or more Sec interns. A head of staff that gets 4+ interns routinely won't be able to meaningfully contribute to the round as a head of staff if they end up needing to train all four of them. Whenever I play CMO, I actually recognize this problem, and pick out a specific person from my medical staff that I know to be experienced, and direct THEM to do the training on my behalf, so that I can actually do my job. 

I'm actually not convinced at all by the arguments that "Seniors shouldn't have more experience than managers", because this doesn't really reflect reality all that much either. I work in Manufacturing, and in my own real life job the "Senior Operators" all have 6 or 8 years or greater of experience, and most of them have more experience working the machines than even their managers. Because the managers have a job of well, being a manager, not being an operator. It's the seniors in my field that are chosen to train new people, because the manager is a busy man, and the senior is more qualified to train people anyway.

**Comment by Nairodian** [comment]
> i like the concept a lot, but i think perhaps it'd be better to just push information to new players more? Things like a more comprehensive guidebook, a first-time popup or notifier informing them on the guidebook and wiki, and maybe even reintroducing something like Tippy the Clown could help disperse information more effectively to players.
> 
> I think any situation where you're directing players to newcomers needs to be handled sensitively, so the best way to avoid that would simply be to give new players resources to learn these things on their own, where they have full control over what information is available to them. If an in-game example is really needed, encouraging them to ask heads about it is probably good enough if paired with other options.
> 
> (please keep the clothes tho they look baller i'll code loadouts just to have them in)

I do still standby being taught in-game is just much more fun, interesting, inviting and memorable to the player. It makes our community feel less unwelcoming if we don't just hand a guy a stack of guides, pat them on the back and wish them luck.

**Comment by Nairodian** [comment]
> What if Seniors/Instructors had a REDUCED chance to be antag, so it still can't be discounted but will be less likely to be an issue for their trainees? Would that still lead to "antagproof favoritism", perhaps not nearly so much

I was thinking of a reduced heavily chance for antag rolls, but not sure what the overall stance on that is, and it still has the chance to ruin some new player's time. I'm not entirely adverse to it however.

**Comment by dmnct** [comment]
I'm also not sure an antag Senior would be THAT ruinous for the trainees. They can't be targets, and the antag would already have access to their department. No great reason to kill them. Roping interns into illegal heists because they don't know any better would be a risky plan with dubious utility. Also probably falls under the "Don't be a dick" rule. I think most likely the interns would just be neglected by a traitor senior, which would still be not great, but not disastrous. An unrelated antag could just as well have an impact on their round as their own trainer, there is no rule against it

**Comment by EmoGarbage404** [comment]
pretty sure interns can be murder targets. i have lead a few sad souls to their doom.

**Comment by deltanedas** [comment]
confirm interns can be targets

**Comment by EmoGarbage404** [comment]
considering that interns aren't optional, making things like this compulsory has a lot of issues. Maybe just give them a  "help me i'm new" hat or something dumb. Making interns a whole separate role with dedicated people trying to help them at all times might end up feeling like a 6 hour long tutorial.

**Comment by Mudsson** [comment]
Just had a thought, what if the senior officer was a Drill Sergeant? Give them a warden like treatment where they don't really leave sec to solve the whole taking experiences from other officers. Not sure how it would work and I'm probably not thinking this through.

**Comment by Mudsson** [comment]
Just had a thought, what if the senior officer was a Drill Sergeant? Give them a warden like treatment where they don't really leave sex to solve the whole taking experiences from other officers. Not sure how it would work and I'm probably not thinking this through.

**Comment by Nairodian** [comment]
> I'm also not sure an antag Senior would be THAT ruinous for the trainees. They can't be targets, and the antag would already have access to their department. No great reason to kill them. Roping interns into illegal heists because they don't know any better would be a risky plan with dubious utility. Also probably falls under the "Don't be a dick" rule. I think most likely the interns would just be neglected by a traitor senior, which would still be not great, but not disastrous. An unrelated antag could just as well have an impact on their round as their own trainer, there is no rule against it

It probably wouldn't be too bad if it was rare, like once per ten rounds. Getting non-antags to do antag things by tricking them _is_ against the rules, and it'd be unlikely for them to ever become a target. Just don't want it to be an overly common thing causing the role to become less about teaching people.

**Comment by Nairodian** [comment]
> considering that interns aren't optional, making things like this compulsory has a lot of issues. Maybe just give them a "help me i'm new" hat or something dumb. Making interns a whole separate role with dedicated people trying to help them at all times might end up feeling like a 6 hour long tutorial.

A fair concern with people trying to be overly helping. As stated with Sloth, interns being removed entirely afterwards isn't out of the picture, in their place a tag on new players' IDs that makes a status symbol show up on their character for senior and heads to see instead?

**Comment by Nairodian** [comment]
> Just had a thought, what if the senior officer was a Drill Sergeant? Give them a warden like treatment where they don't really leave sec to solve the whole taking experiences from other officers. Not sure how it would work and I'm probably not thinking this through.

Security is definitely the least issue out of all of these, but I'd rather make things acceptable for all of them and get them all in at once.

**Comment by Emisse** [comment]
100% these need to be allowed to antag so they arent put on pedestal of 2nd department head

**Comment by Emisse** [comment]
also it needs to be communicated in the description that players in this roles primary focus should be assisting players in their department over anything else so its not just an extra job slot

**Comment by potato1234x** [comment]
the sprites are eye candy but sec doesnt need more roles, all the others are a good idea tho

**Comment by UbaserB** [comment]
> the sprites are eye candy but sec doesnt need more roles, all the others are a good idea tho

warden doesnt really count as a second head since they take care of prisoners; other than wardenloose of course. Security has a large number of newbies flocking in and bad sec members are at times really annoying and frustrating. HoS doesnt have the luxury to teach newbies as theres a lot of stuff to deal with.

**Comment by GoodWheatley** [comment]
I think it’s a better idea to properly implement a mentorhelp system instead of creating effectively auxiliary heads for every department and turning intern roles into one long tutorial. As someone who prefers to figure stuff out on my own, I would absolutely despise playing as an intern if it meant I had someone hovering around me the entire time telling me exactly what to do.

**Comment by dmnct** [comment]
I don't think the addition of trainers would automatically mean interns are now forced into training, just like how I never had a Head INSIST on training me if I was playing intern, if they offered without me asking about it and I was not interested. 
But having the trainers be included in the job system would make it simple to regulate how many of them there are, and how they are "vetted" via the systems we already have in place - play time and job bans
If the senior has no one to train they could just do their department job or offer training for passengers/anyone interested

**Comment by GoodWheatley** [comment]
> But having the trainers be included in the job system would make it simple to regulate how many of them there are, and how they are "vetted" via the systems we already have in place - play time and job bans

I agree with Emo on this; a 60 hour time requirement is only going to filter out the worst players at best. Playing 60 hours in a department only guarantees that the player knows what they’re doing, not that they’ll be willing to teach other players what they know. And expecting admins to simply job-ban mentors who aren’t good at their jobs is likely just going to create *more* admin workload in the long run.

Instead of selectively blacklisting mentors who are ignoring interns, which would require monitoring mentors throughout the course of the round to ensure they’re doing their job, it would be much easier to just selectively *whitelist* experienced players who have the knowledge and temperament to be good teachers, ergo why I suggested that mentorhelp should be implemented.

**Comment by UbaserB** [comment]
> I think it’s a better idea to properly implement a mentorhelp system instead of creating effectively auxiliary heads for every department and turning intern roles into one long tutorial. As someone who prefers to figure stuff out on my own, I would absolutely despise playing as an intern if it meant I had someone hovering around me the entire time telling me exactly what to do.

i personally loved this when i was learning how the game worked, it felt really IC and comfortable, and they were good at teaching me.

**Comment by Nairodian** [comment]
Fuck it, if the antags suck, changes can be made.

**Comment by Emisse** [comment]
how many job slots of these should be present per map, should these replace existing job slots, etc

**Comment by Nairodian** [comment]
> also it needs to be communicated in the description that players in this roles primary focus should be assisting players in their department over anything else so its not just an extra job slot

image

**Comment by Nairodian** [comment]
> how many job slots of these should be present per map, should these replace existing job slots, etc

_My_ thoughts were one of each per map, and it could replace the spot of an intern or main role, though as the headmapper that choice is ultimately up for you.

**Comment by Emisse** [comment]
ok ill figure it out

**Comment by UbaserB** [comment]
do they still require 60 hours or did you guys decide on the whitelist?

**Comment by Nairodian** [comment]
Still 60 hours.

**Comment by JoeHammad1844** [comment]
MFW they forget the medical beret
